src/components/MoneyReportHeader.js

```diff
@@ -126,6 +129,17 @@ function MoneyReportHeader({session, personalDetails, policy, chatReport, report
                         />
                     </View>
                 )}
+                {shouldShowSubmitButton && !isSmallScreenWidth && (
+                    <View style={styles.pv2}>
+                        <Button
+                            success
+                            medium
+                            text={translate('common.submit')}
+                            style={[styles.mnw120, styles.pv2, styles.pr0]}
+                            onPress={() => IOU.submitReport(moneyRequestReport)}
+                        />
+                    </View>
```

Comment:
Same comment as here https://github.com/Expensify/App/pull/28918#discussion_r1348675134 different appearance based on who is looking at the report